tracer/src/Datadog.Trace/Configuration/TransportSettings.cs

```diff
@@ -0,0 +1,190 @@
+// <copyright file="TransportSettings.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.IO;
+using Datadog.Trace.Agent;
+using Datadog.Trace.Vendors.StatsdClient.Transport;
+
+namespace Datadog.Trace.Configuration
+{
+    /// <summary>
+    /// Contains transport settings.
+    /// </summary>
+    public class TransportSettings
```

Comment:
OTEL uses the term Exporter for emitting information out of process. What if we did something similar and renamed this to ExporterSettings or AgentSettings? The name TransportSettings is ok to me but I think it can be more descriptive